geoh5py/objects/surveys/electromagnetics/base.py

```diff
@@ -178,6 +178,22 @@ def channels(self, values: list | np.ndarray):
 
         self.edit_metadata({"Channels": values})
 
+    @property
+    def components(self) -> dict | None:
```

Comment:
**suggestion (tests):** have unit test for this new property
especially because it is an API entry point which does not seem used in the code base

Reply:
the unittest was added to the tem_test

Reply:
👍
I missed it sorry